### PR TITLE
Add missing french translations

### PR DESCRIPTION
**Pull request by francois-ferrandis**
I chose to reorder the entries of `fr.yml` so that they match the order of the entries in `en.yml`, is that a reasonable practice?

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -73,14 +74,18 @@ fr:
...
+      author_type: "Profil de l'auteur"
       body: "Corps"
       author: "Auteur"
       title: "Commentaire"
       add: "Ajouter un commentaire"
+      delete: "Supprimer ce commentaire"
+      delete_confirmation: "Êtes-vous sûr de vouloir supprimer ce commentaire ?"
       resource: "Ressource"
-      resource_type: "Type de ressource"
-      author_type: "Profil de l'auteur"
```
Comment (timoschilling) [inline config/locales/fr.yml]: why did you remove this keys?

Comment (deivid-rodriguez) [inline config/locales/fr.yml]: He didn't, he just reordered them in bed7a64e1347914d27f0a742b1f047f484af0d75 to match the order in the English version. I'm :+1: for the change!

Comment (timoschilling) [inline config/locales/fr.yml]: sorry didn't see that

Comment (deivid-rodriguez) [inline config/locales/fr.yml]: No problem! 😉
